content/2015.07.whistler-use-cases-en.rst

```diff
@@ -0,0 +1,171 @@
+
+Cut The Rope
+------------
+
+Kinto is likely to be used to synchronise the configuration settings and scores of the very popular game `Cut The Rope <https://marketplace.firefox.com/app/cut-the-rope/>`_. It'll be good to have a practical use-case on such a visible platform.
+
+Syncto
+------
+
+`Firefox Sync <https://docs.services.mozilla.com/storage/apis-1.5.html>`_ is what enables synchronisation between Firefox (favourites, extensions, browser history, etc.) and many other targets in an encrypted fashion. The current Javascript client implementation is relatively complex and, frankly, somewhat dated (the proposed re-write has yet to occur).
+
+We'd like to implement a bridge between *Kinto* and *Firefox Sync* via *Kinto.js* - which is more simple and more modern than the existing reference client - in order to store synced data in IndexedDB. On the server side, we feel that this shouldn't be too hard since our project is itself inspired by Sync's proven protocol model; from a client perspective, most of the work will be in plumbing crypto into BrowserID authentication.
+
+Alexis jumped at the chance to start work on a `Python Firefox Sync client <https://github.com/mozilla-services/syncclient>`_ which will serve as the base for any future service.
+
+Cloud Storage
+-------------
+
+Eden Chung and Sean Lee did a presentation on their advancements regarding the integration of remote storage services (notably DropBox and Baidu Yun) in Firefox OS. Their current proof-of-concept is based on `FUSE <https://en.wikipedia.org/wiki/Filesystem_in_Userspace>`_.
+
+We were also thinking about introducing the idea of attachments in Kinto by implementing the `remote storage specification <https://tools.ietf.org/html/draft-dejong-remotestorage-05>`_; however, for the moment our existing use-cases just aren't calling for it.
+
+Coming Soon
+-----------
+
+In addition to the work that we've already done on the server side, we're going to have to deal with concurrency management in our Javascript client in order to allow simultaneous writes and background synchronisation.
+
+We are always looking for feedback - and, of course, code contributions - on both the `server <https://github.com/mozilla-services/kinto/>`_ and `client <https://github.com/mozilla-services/kinto.js/>`_!
+
+.. image:: {filename}/images/whistler-cloud-storage.jpg
+    :alt: Firefox OS Cloud Storage Presentation - CC0
+    :align: center
+
+Firefox Application Content
+===========================
+
+The Firefox development and release process is currently based on a six-month cycle. One of the objectives is to decouple certain specific content elements (such as safety rules, dictionaries and translations, etc.) from these relatively long cycles [#]_.
```

Comment:
In fact, the original French text states "six mois"; I will adjust it there as well.